### PR TITLE
Feature/3 rename contracts

### LINKED ISSUE #3: Rename mainnet contracts
Need to rename mainnet contraxts from neofs* to frostfs*

### PR DESCRIPTION
**Pull request by KirillovDenis**
close #3

### CODE REVIEW

**Review by carpawell — COMMENTED** [review]
1. Contract names in `config.yml`;
2. Archive target in `Makefile`;
3. `README` appearances.

Not sure if these ones should be changed in that PR.

**Comment by fyrchik** [comment]
What about `neofsid`?

**Comment by KirillovDenis** [comment]
> What about `neofsid`?

As I understand NeoFSID contract is a contract deployed in NeoFS sidechain, but this PR is about mainchain

**Comment by fyrchik** [comment]
Yeah, I mean can we do it here too?

**Comment by realloc** [comment]
If we are ok with renaming everything at once, let's rename everything.

File: README.md [hunk 1]
```diff
@@ -79,17 +79,17 @@ ok      github.com/TrueCloudLab/frostfs-contract/tests       0.462s
...
+|         v0.14.x          |                                                                      [v2.11.0](https://github.com/nspcc-dev/neofs-api/releases/tag/v2.11.0)                                                                      |
+|         v0.15.x          |                                  [v2.11.0](https://github.com/nspcc-dev/neofs-api/releases/tag/v2.11.0), [v2.12.0](https://github.com/nspcc-dev/neofs-api/releases/tag/v2.12.0)                                  |
+|         v0.15.x          |                                  [v2.11.0](https://github.com/nspcc-dev/neofs-api/releases/tag/v2.11.0), [v2.12.0](https://github.com/nspcc-dev/neofs-api/releases/tag/v2.12.0)                                  |
+|         v0.16.x          |                                                                      [v2.14.0](https://github.com/nspcc-dev/neofs-api/releases/tag/v2.14.0)                                                                      |
```
Comment (alexvanin) [inline README.md]: Not sure if this table is relevant for frostfs.

**Review by alexvanin — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (realloc) [reply README.md]: Agree. Let us just drop it.

**Review by alexvanin — APPROVED** [review]

**Pull request merged**